app/views/setup/settings/_mirror.html.slim

```diff
@@ -0,0 +1,38 @@
+- certificate = @suse_registry_mirror.certificate.present? ? @suse_registry_mirror.certificate.certificate : ''
+
+= form.fields_for :suse_registry_mirror do |f|
+  .suse-mirror-panel.panel.panel-default
+    .panel-heading.clearfix
+      h3.panel-title SUSE registry mirror
+      .btn-group.btn-group-sm.btn-group-toggle.pull-right data-toggle="buttons"
+        = label_tag :mirror_toggle, nil, class: "enable-suse-registry-mirror btn btn-default #{'btn-primary active' if @suse_registry_mirror_enabled}", data: {toggle: "collapse", target: ".suse-mirror-panel-body"}
+          = form.radio_button :suse_registry_mirror_enabled, "enable", checked: @suse_registry_mirror_enabled
+          | Enable
+        = label_tag :mirror_toggle, nil, class: "disable-suse-registry-mirror btn btn-default #{'btn-primary active' unless @suse_registry_mirror_enabled}", data: {toggle: "collapse", target: ".suse-mirror-panel-body"}
+          = form.radio_button :suse_registry_mirror_enabled, "disable", checked: !@suse_registry_mirror_enabled
+          | Disable
+
+    .suse-mirror-panel-body.panel-collapse.collapse class="#{'in' if @suse_registry_mirror_enabled}"
+      .panel-body
+        .form-group.mirror-url-group
+          = f.label :url, "URL"
+          .input-group
+            = f.url_field :url, value: @suse_registry_mirror.url, class: "form-control", placeholder: "https://suse-mirror.local.lan:5000"
+            span class="input-group-addon"
+              a data-toggle="tooltip" data-placement="left" title="This is the URL of the local registry configured to be a mirror of the SUSE container registry. Use the http:// or https:// prefixes depending on how the local mirror has been deployed. Include also the port number if the mirror isn't using a standard one, eg: https://suse-mirror.local.lan:5000"
+                i class='glyphicon glyphicon-info-sign'
+          span.help-block.invalid-url.hide This is not a valid URL. Please try something like https://registry.local
+          span.help-block.invalid-insecure.hide You are using an insecure mirror address for a secure remote registry
```

Comment:
Can you change the message to be: `<strong>Security warning:</strong> you are using...`.